server/service/src/invoice/inbound_shipment/add_from_master_list.rs

```diff
@@ -107,7 +108,7 @@ fn generate(
         .map(|master_list_line| master_list_line.item_id)
         .collect();
 
-    generate_empty_invoice_lines(ctx, &invoice_row, items_ids_not_in_invoice)
+    generate_empty_invoice_lines(connection, ctx, &invoice_row, items_ids_not_in_invoice)
```

Comment:
@noel-yeldos FYI `connection` is available from the `ServiceContext` object `ctx`, to access the connection you can just do `ctx.connection`, so adding the extra parameter here isn't necessary...

Reply:
@jmbrunskill ah good to know! I'll fix as sidequest on current PR :)